rdmo/projects/assets/js/interview/utils/value.js

```diff
@@ -39,28 +39,33 @@ const initValues = (sets, values, element, setPrefix) => {
     setPrefix = ''
   }
 
+  // loop over all sets of the current set prefix and over all questions
   sets.filter((set) => set.set_prefix === setPrefix).forEach((set) => {
     element.elements.filter((e) => (e.model === 'questions.question')).forEach((question) => {
```

Comment:
do you maybe wanna use `.flatMap` here instead of nesting?
```js
  .filter((set) => set.set_prefix === setPrefix)
  .flatMap((set) =>
    element.elements.filter((e) => e.model === 'questions.question').map((question) => ({ set, question }))
  )
  .forEach(({ set, question }) => {
```

Reply:
I did not know about `flatMap`. I think I will keep the 2 loops here, since it was not changed in this PR and I want to minimize the risk to break things. But I will use it in the future. I also think the nested loops are a bit better to read.